src/Exporter/DataItemToElementMapper.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace SMW\Exporter;
+
+use SMW\Exporter\Element\ExpResource;
+use SMW\Exporter\Element\ExpNsResource;
+use SMW\Exporter\Element\ExpElement;
+use SMW\Exporter\Element\ExpLiteral;
+use SMWExporter as Exporter;
+use SMWDataItem as DataItem;
+use SMWDITime as DITime;
+use RuntimeException;
+
+/**
+* @license GNU GPL v2+
+* @since 2.2
+*
+* @author mwjames
+* @author Markus Krötzsch
+*/
+class DataItemToElementMapper {
+
+	/**
+	 * @var array
```

Comment:
`@var Closure[]`

Though actually you probably should be using `callable`, which includes `Closure`, though also allows for other things